packages/eas-build-job/src/common.ts

```diff
@@ -115,7 +115,10 @@ export interface Cache {
   disabled: boolean;
   clear: boolean;
   key?: string;
-  cacheDefaultPaths: boolean;
+  /**
+   * @deprecated We don't cache anything by default anymore.
+   */
+  cacheDefaultPaths?: boolean;
   customPaths: string[];
```

Comment:
Since we don't have default paths now, it'd make more sense to rename `customPaths` to `paths`. You can either change this in this PR or in a follow-up one.  We'll have to implement backward compatibility for the old field.

Reply:
I will leave it as a follow-up task on which @radoslawkrzemien can work. What do you think?

Reply:
Working on it

Reply:
https://linear.app/expo/issue/ENG-8120/rename-cachecustompaths-to-cachepaths-in-easjson-and-job-schema